ui/app/styles/core/table.scss

```diff
@@ -86,6 +86,30 @@
       overflow: hidden;
       text-overflow: ellipsis;
       white-space: nowrap;
+
+      &.is-hover-overflowable {
+        position: relative;
+
+        .hover-overflow {
+          display: none;
+          position: absolute;
+          top: 0;
+          left: 0;
+          padding: calc(1.25em - 1px) 1.5em;
+        }
+
+        &:hover {
+          color: transparent;
+          overflow: visible;
+
+          .hover-overflow {
+            color: $text;
+            display: block;
+            background-color: rgba(255, 255, 255, 0.9);
```

Comment:
I'd prefer for this to be solid `$white`. As is, the slight transparency just makes the text harder to read.

Also, for future reference, you can do `rgba($white, 0.9)` and it works as expected.

Reply:
oh nice, thanks, I should have known, Sass 😍

yes, I like `.is-revealable`, and changed `hover-overflow` to `revealed` 😀